### ISSUE TITLE
During training, the map results of some configurations are always 0

### ISSUE THREAD

**Issue by jiewjwang** [Sompote/DinoV3-YOLO-Segment#6]
When I was doing a training experiment on the crack dataset, I used the configuration you recommended, but the map was always 0（After running forty or fifty epochs, the map is still 0）. I would like to ask what went wrong.  
The configuration is as follows：python train_yolov12_segmentation.py     --data ../datasets/crack-seg.yaml     --model-size l     --use-dino     --dino-variant vitl16     --dinoversion v3     --integration triple     --optimizer AdamW     --lr 0.001     --momentum 0.9     --weight-decay 0.01     --epochs 300     --batch-size 4     --patience 50     --name crack_triple_dino
If you can answer, thank you very much！

**Comment by Sompote** [comment]
Upon further consideration, the value is non-zero but exhibits a small magnitude (approximately 10⁻⁵). This behavior appears to be dataset-dependent. If the issue cannot be resolved from your end, I would appreciate access to your dataset to perform a comprehensive analysis and identify the underlying cause.

**Comment by jiewjwang** [comment]
Hello, thank you for your answer. The dataset I use here is the crack-seg dataset.

**Comment by Sompote** [comment]
Please try another segment dataset. If it not work, please tell me.

**Comment by xpwindells** [comment]
I also encountered this problem. If Dino selects the vit-large model, the mIoU value of the validation set remains at 0.
command:
python train_yolov12_segmentation.py \
    --data data.yaml \
    --model-size l \
    --use-dino \
    --dino-variant vitl16 \
    --dinoversion v3 \
    --integration triple \
    --optimizer AdamW \
    --lr 0.001 \
    --momentum 0.9 \
    --weight-decay 0.01 \
    --epochs 300 \
    --batch-size 4 \
    --patience 50 \
    --name crack_triple_dino
